src/Razor/src/Microsoft.VisualStudio.LiveShare.Razor/Guest/ProjectSnapshotSynchronizationService.cs

```diff
@@ -14,7 +12,7 @@
 
 namespace Microsoft.VisualStudio.LiveShare.Razor.Guest
 {
-    internal class ProjectSnapshotSynchronizationService : ICollaborationService, IAsyncDisposable
+    internal class ProjectSnapshotSynchronizationService : ICollaborationService, IAsyncDisposable, System.IAsyncDisposable
```

Comment:
Interesting why now?

Reply:
There's an analyzer which ask you to implement `System.IAsyncDisposable` if you implement the other one. I figured I may as well fix it while in here since this is a "Do the right thing for Async" PR.